ui/components/app/account-menu/index.scss

```diff
@@ -100,8 +100,9 @@
   & &__lock-button {
     @include H7;
 
-    padding: 3.5px 5px;
-    width: 59px;
+    padding: 0.25rem 0.5rem;
+    min-width: 59px;
+    width: auto;
```

Comment:
would `width: 100%` be better?

Reply:
100% will stretch it left to the label
![image](https://user-images.githubusercontent.com/5180700/161403341-849bbda5-29c5-428c-9a80-846008fbf30e.png)
Nah

Reply:
of course we can add some margin between
![image](https://user-images.githubusercontent.com/5180700/161403445-953432e6-c46f-4841-87b2-fbffe362df84.png)
it would look okay